repos/exawind/packages/nalu-wind/package.py

```diff
@@ -51,8 +51,6 @@ class NaluWind(CmakeExtension, bNaluWind, ROCmPackage):
     depends_on("trilinos gotype=long")
     depends_on("openfast@develop,fsi+netcdf+cxx", when="+fsi")
 
-    patch("rocm1.patch")
```

Comment:
should this be kept for certain versions?

Reply:
No, it's been applied in https://github.com/Exawind/nalu-wind/pull/1255